lib/version_utils.pm

```diff
@@ -194,6 +194,19 @@ sub check_version {
     croak "Unsupported version parameter for check_version: '$query'";
 }
 
+=head2 is_microos
+
+Check if product is SUSE MicroOS
+Version: 5.X, 6.X
+=cut
+sub is_microos {
+    my $distri  = get_var('DISTRI');
+    my $version = get_var('VERSION');
+    return 0 unless $distri && $distri =~ /microos/;
+    return 1 if $version =~ "^[56].";
```

Comment:
`$distri`  is really variable which has no point to check if it is exists or not. because when `$distri`  is not defined you just simply not able to reach main.pm :) . so at least you can remove `$distri &&`  
Also basically you need to return 1 when distri is equal to `microos` **and** version starts from 5 or 6 . This is direct translation of this sentence : 
```suggestion
   return   $version =~ "^[56]." &&  $distri =~ /microos/;
```